### PR TITLE
Auto shooting command group

### PR DESCRIPTION
**Pull request by Silvery-Sheen**
Pull Request Checklist:
- [x] You are able to understand what the code is doing on a basic level with the use of comments
- [x] Code is formatted in a way that isn't confusing
- [ ] builds cleanly
- [ ] builds *to the robot* cleanly
- [ ] Code has been tested on the robot/roadkill/other mechanical setup and completes the task it needs to complete

### CODE REVIEW

**Review by BenWertz — APPROVED** [review]

File: src/main/java/frc/robot/autoCommands/autoDriveToPoint.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  public boolean isFinished() {
+    return false;
+  }
+
+  public double returnAngleToTurn(double startY, double startX, double endY, double endX) {
+    // Calculates the length of the legs of a triangle made with the robot path as the hypotenuse
+    double XLength = Math.abs(endX - startX);
+    double YLength = Math.abs(endY - startY);
+
+    // Uses the inverse tangent to calculate the measure of the angle in degrees that the robot should turn
```
Comment (122004) [inline src/main/java/frc/robot/autoCommands/autoDriveToPoint.java:52]: Nice, succinct commenting.

File: src/main/java/frc/robot/autoCommands/autoShootingMidOfField.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class autoShootingMidOfField extends SequentialCommandGroup {
+
+  /**
+   * Creates a new autoShootingAlignedWithTarget.
+   */
+  
+  public autoShootingMidOfField() {
+    super();
+    new autoShootingCommandGroup(0.762, 1.6764, 0, 0, 0);
```
Comment (122004) [inline src/main/java/frc/robot/autoCommands/autoShootingMidOfField.java:23]: Where did these values come from?

Comment (Silvery-Sheen) [reply src/main/java/frc/robot/autoCommands/autoShootingMidOfField.java:23]: They need to be tested an finetuned

Comment (122004) [reply src/main/java/frc/robot/autoCommands/autoShootingMidOfField.java:23]: All right then.

**Review by 122004 — APPROVED** [review]

**Pull request merged**